src/View/Helper/PaginatorHelper.php

```diff
@@ -1121,4 +1121,31 @@ public function implementedEvents()
     {
         return [];
     }
+
+    /**
+     * Dropdown array for Select limit
+     *
+     * @param array $limits This is array of options.
+     * @param array $default Default limit for option selecting. Default is Paginator limit.
+     *
+     * @return string html output.
+     */
+    public function showLimit($limits = [], $default = 0)
```

Comment:
limit() is already used or confusing, may be some other name can be suggested please. I use showLimit because to mean showing limit options. Thanks for suggestion.